docs/experimental-features.md

```diff
@@ -43,6 +43,11 @@ experimental, default-disabled.
 run your daemon with the `--enable-pubsub-experiment` flag. Then use the
 `ipfs pubsub` commands.
 
+### gossipsub
+
+You can enable the new experimental gossipsub protocol via configuration:
```

Comment:
I'd add a short line explaining the difference between the default floodsub and gossipsub.

Reply:
hrm, maybe, although that's a topic in itself. there will be a gossipsub spec soon [TM] and perhaps we could link to that instead.

Reply:
For now, I'd just go with:

>  Gossipsub is a new, experimental routing protocol for pubsub that should waste less bandwidth than the current default routing protocol (floodsub). It's backwards compatible with floodsub so enabling this feature shouldn't break compatibility with existing IPFS nodes.

That should tell users everything they need to know (for now).

Reply:
done.